components/feature/qr/src/main/java/mozilla/components/feature/qr/QrFragment.kt

```diff
@@ -13,6 +13,10 @@
  * See the License for the specific language governing permissions and
  * limitations under the License. */
 
+// Deprecation of getDefaultDisplay() and getSize() will be handled in https://github.com/mozilla-mobile/android-components/issues/8518
+// Deprecation of createCaptureSession() will be handled in https://github.com/mozilla-mobile/android-components/issues/8510
+@file:Suppress("DEPRECATION")
+
```

Comment:
It will be better to do specific deprecation, we could hide future deprecations

Reply:
These annotations will be removed in the PR for fixing the specific deprecation, they serve only as a way to pass the first upgrade build and as a link to the specific issue. For files where multiple deprecations (or multiple instances of the same) I thought it will be easier to track than having the annotation in many places inside the file.

Reply:
Agree :), the issue is that while they are being worked new code could be introduced with new deprecations, it will be noticed only when global deprecation is removed, if we have individual deprecation developers will be aware of new possible and could fix them right away.